templates/help.html

```diff
@@ -0,0 +1,30 @@
+{% extends "base.html" %}
+{% load static %}
+{% block extra_css %}
+{% endblock extra_css %}
+{% block title %}Help{% endblock title %}
```

Comment:
Just out of curiosity, should the title be "Help" or something like "Help | Solved"?
@chenyx512 you got any preferences we should stick to?

Reply:
That is your choice, or you could as Jenny for design specs, the "Help | Solved" seems to be @Miclin1024 's style

Reply:
I think we ought to be consistent with this.
I will consult Jenny.